setup.py

```diff
@@ -28,6 +28,7 @@
     'pyramid_mako>=0.3.1', # lazy configuration loading works
     'repoze.lru',
     'Pygments',
+    'ipaddress;python_version<"3.3"',
```

Comment:
This particular syntax is not supported for wheels right now. Please use the environment markers as an extra:

https://bitbucket.org/pypa/wheel/issues/172/wheel-doesnt-support-environment-markers#comment-30414396